manifests/e/EclipseAdoptium/Temurin/18/JDK/18.0.2.1.1/EclipseAdoptium.Temurin.18.JDK.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.18.JDK
+PackageVersion: 18.0.2.1.1

```

```suggestion
PackageVersion: 18.0.2.101
```
This must match Control Panel; you will also need to update the version folder to match PackageVersion.